haystack/pipelines/base.py

```diff
@@ -216,6 +280,53 @@ def list_pipelines_on_deepset_cloud(
         pipeline_config_infos = list(client.list_pipeline_configs())
         return pipeline_config_infos
 
+    @classmethod
+    def save_to_deepset_could(
+        cls,
+        query_pipeline: BasePipeline,
+        index_pipeline: BasePipeline,
+        pipeline_config_name: str,
+        workspace: str = "default",
+        api_key: Optional[str] = None,
+        api_endpoint: Optional[str] = None,
+        overwrite: bool = False,
+    ):
+        """
+        Saves a Pipeline config to Deepset Cloud defining the individual components and how they're tied together to form
+        a Pipeline. A single config must declare a query pipeline and a index pipeline.
```

Comment:
Out of curiosity, why you need both? Can't a config contain only a query pipeline?

Reply:
DC currently needs both. I think it's because pipeline_config_name always is the underlying index name, too. So without an indexing pipeline, there wouldn't be any chance to get documents. Ok you might want to use DCDocumentStore to index them manually, but DC's config validation will complain about the missing indexing pipeline anyway.